server.js

```diff
@@ -113,7 +116,26 @@ app.get('/api/v1/playlists', (request, response) => {
     response.status(500).json({ error });
   });
 });
+app.get('/api/v1/playlists/:id/songs', (request, response) => {
+  let playlistResponse;
+  let playlistId = request.params.id;
 
+  database('playlists').where('id', playlistId).select(['id', 'playlist_name'])
+  .then(playlists => {
+    if(playlists.length) {
+      playlistResponse = playlists[0];
+      database('playlist_songs').where('playlist_id', playlistId)
+        .select(['songs.id', 'name', 'artist_name', 'genre', 'song_rating'])
```

Comment:
I wonder if putting these column names is giving you any kind of odd behavior?  They do not need to go into an array structure like this.